backend/src/routes/user.ts

```diff
@@ -422,6 +435,37 @@ router.get(
   }
 );
 
+// type UpdateUserType = Infer<typeof UpdateUserCake>
+router.patch(
+  "/user/:userId",
+  [verifyAuthToken],
+  validateReqBodyWithCake(UpdateUserCake),
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      console.log("Starting patch");
```

Comment:
Let's just keep this log statement and instead of logging "starting patch" maybe something like "Updating FCM token" so its more clear for future debugging.